manifests/m/Microsoft/PIX/24.3.8.1/Microsoft.PIX.installer.yaml

```diff
@@ -0,0 +1,24 @@
+# Created with komac v2.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Microsoft.PIX
+PackageVersion: 24.3.8.1
```

Comment:
Please update the versioning scheme to be consistent with PIX. The correct version number is 2403.08

Reply:
@sfreed141 - To do that all other versions of PIX will need to be removed and re-added to the repository to ensure the correct sorting order.  Using this version scheme is currently necessary as that is what PIX writes to Add and Remove Programs

![image](https://github.com/microsoft/winget-pkgs/assets/12611259/d605ca32-e6e3-4c7d-a284-dc3b36e5d22d)

Reply:
Ah okay, thanks for the explanation!

Reply:
> [!NOTE]
>
> Because of the following reason, this change will not be applied.
>
> - See https://github.com/microsoft/winget-pkgs/pull/146944#discussion_r1544686470